### DISCUSSION TITLE
Enable GitHub Discussions.

### DISCUSSION THREAD

**Discussion by daelvn** [keplerproject/luarocks#1269] [Ideas]
GitHub recently got a new feature called [Discussions](https://docs.github.com/en/discussions), which have to be enabled manually for every repository.

I think it would be useful to have Discussions enabled on this repository to ask how certain things are done properly with LuaRocks (without taking up an issue), talking about features, solutions, showcases or maybe even discussion the possible future of LuaRocks.

**Comment by DarkWiiPlayer** [comment]
I second this idea 😀

**Comment by daurnimator** [comment]
Done :)
